nuxt.config.ts

```diff
@@ -153,15 +153,18 @@ export default defineNuxtConfig({
         quality: 75,
         screens: {
             xs: 375, // override size to match our breakpoints
-            xl: 1440, // override size to match our breakpoints
+            sm: 480, // overridden
+            vl: 1280, // overridden
+            xl: 1440, // overridden
+            xxl: 1600, // overridden
```

Comment:
what do you mean by 'overriden'? These breakpoints already exist (defined by Nuxt image module)

Reply:
these breakpoints aren't iso with ours
![image](https://github.com/user-attachments/assets/7760b5b8-fdb2-4b0a-8b0f-cf1087db15cc)